app/functions

```diff
@@ -0,0 +1,35 @@
+# -*-Shell-script-*-
+#
+# This file contains functions to be used by most or all
+# shell scripts in the /app directory.
+#
+
+debug() {
+  if [ -n "${1}" ] && [ ${DEBUG} -eq 1 ]; then
+    echo "${1}"
+  fi
+}
+
+rclone_aws() {
+  rclone \
+    --s3-provider=AWS \
+    --s3-region="${AWS_DEFAULT_REGION}" \
+    --s3-location-constraint="${AWS_DEFAULT_REGION}" \
+    --s3-access-key-id=${AWS_ACCESS_KEY_ID} \
+    --s3-secret-access-key=${AWS_SECRET_ACCESS_KEY} \
+    ${RCLONE_ADD_PARAMS} $@
+}
```

Comment:
as we discussed, when a iam role is present we don' t need any other credential, i think we can avoid them totally, because we should ever assume that the credentials (if needed) are available like env vars.

```
rclone_aws() {
  rclone ${RCLONE_ADD_PARAMS} $@
}
```

We should have just this one basically, we can get rid also of the `--s3-provider` because it's the standard one

Reply:
I have simplified with the minimum required argument:
```bash
--s3-region="${AWS_DEFAULT_REGION}" \
--s3-location-constraint="${AWS_DEFAULT_REGION}" \
--s3-env-auth
```